modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -129,6 +131,15 @@
  */
 @SuppressWarnings({"TooBroadScope"})
 public abstract class GridCacheMapEntry extends GridMetadataAwareAdapter implements GridCacheEntryEx {
+    /** */
+    public static final GridCacheAtomicVersionComparator ATOMIC_VER_COMPARATOR = new GridCacheAtomicVersionComparator();
+
+    /** Entry lock time awaiting. */
+    public static final long ENTRY_LOCK_TIMEOUT = getLong("ENTRY_LOCK_TIMEOUT", 1000);
```

Comment:
I don't think that we should extract this to IgniteSystemProperties. However, it's a good practice to extract such string literals ("ENTRY_LOCK_TIMEOUT") as constant fields.